documentation/faqs.md

```diff
@@ -7,7 +7,7 @@ It has been tested on 1000 node Kubernetes cluster so far. Please refer to this
 
 
 ## Can vSphere Cloud Provider support Kubernetes Cluster spanning across multiple vCenters, ESXi Clusters and Datacenters?
-With Kubernetes version 1.9.x vSphere cloud provider supports Kubernetes cluster spanning across multiple ESXi clusters, vSphere Datacenters and vCenters. All Kubernetes node VMs must have access to shared storage.
+With Kubernetes version 1.9.x vSphere cloud provider supports Kubernetes cluster spanning across multiple ESXi clusters, vSphere Datacenters and vCenters. All Kubernetes node VMs must have access to shared storage. Also see [zone support](/vsphere-storage-for-kubernetes/documentation/zones.html) for using storage that is shared only by node VMs within a zone. Zone support does not work across multiple vCenters.
```

Comment:
Can we mode declaration about zone support with another Question?
We might confuse user with this statement here, as from 1.9.0 we started supporting multi vCenter, but zone support is added in 1.13.x ...

Reply:
Alright. I will remove this mention and leave the zone support in a separate faq.